source/components/defining-a-component.md

```diff
@@ -5,6 +5,10 @@ create a `components/blog-post` template.
 **Note:** Components must have at least one dash in their name. So `blog-post` is an acceptable name, so is `audio-player-controls`, but `post` is not. This prevents clashes with current or future HTML element names, and
 ensures Ember detects the components automatically.
 
+```bash
+ember generate component blog-post
+```
+
 A sample component template would look like this:
```

Comment:
We should decide on https://github.com/emberjs/guides/issues/519 and introduce generators at the top of the guides and be done with it.